rotkehlchen/data_import/importers/binance.py

```diff
@@ -771,42 +792,39 @@ def _process_binance_rows(
             else:
                 rows_without_multiple = rows_without_single
 
+            skipped_count += len(rows_without_multiple)
             if len(rows_without_multiple) > 0:
-                skipped_rows.append([timestamp, rows_without_multiple])
+                ignored_rows.append([timestamp, rows_without_multiple])
 
         skipped_nontrade_rows = []
         skipped_trade_rows = []
-        for skipped_rows_group in skipped_rows:
+        for skipped_rows_group in ignored_rows:
             cur_operations = {el['Operation'] for el in skipped_rows_group[1]}
             if cur_operations.issubset(BINANCE_TRADE_OPERATIONS):
                 skipped_trade_rows.append(skipped_rows_group)
             else:
                 skipped_nontrade_rows.append(skipped_rows_group)
         total_found = sum(stats.values())
-        skipped_count = 0
-        for _, rows in skipped_rows:
-            skipped_count += len(rows)
+        ignored_count = 0
+        for _, rows in ignored_rows:
+            ignored_count += len(rows)
         log.debug(f'Skipped Binance trade rows: {skipped_trade_rows}')
         log.debug(f'Skipped Binance non-trade rows {skipped_nontrade_rows}')
         log.debug(f'Total found Binance entries: {total_found}')
-        log.debug(f'Total skipped Binance csv rows: {skipped_count}')
+        log.debug(f'Total skipped Binance csv rows: {ignored_count}')
         log.debug(f'Binance import stats: {[{type(entry_class).__name__: amount} for entry_class, amount in stats.items()]}')  # noqa: E501
-        if skipped_count > 0:
-            self.db.msg_aggregator.add_warning(
-                f'Skipped {skipped_count} rows during processing binance csv file. '
-                f'Check logs for details',
-            )
+
+        return skipped_count
 
     def _import_csv(self, write_cursor: DBCursor, filepath: Path, **kwargs: Any) -> None:
         """
         Group and process binance CSV entries. May raise:
         - InputError
         """
+        self.name = 'Binance'
```

Comment:
this should be assigned at initialization right? also in other importer too

Reply:
Most of the importers do not currently have an `__init__` function. Is it really worth adding all of this just to be able to set the name at initialization?

```
from typing import TYPE_CHECKING
...
if TYPE_CHECKING:
    from rotkehlchen.db.dbhandler import DBHandler
...

    def __init__(self, db: 'DBHandler') -> None:
        super().__init__(db=db)
        self.name = 'Binance'
```
If it really should be done that way I can add an init function to all the importers, but I assumed it would be better to simply define it at the beginning of the `_import_csv` function.

Reply:
I mean... we should keep it consistent, and also the holy bible of OOPs and DRY says that lol
I suggested a cleaner way do to it in another comment